api/index.js

```diff
@@ -1 +1,30 @@
-module.exports = { v1: require('./v1') };
+var config = require('./config');
+var logger = require('./logging');
+
+var v1 = require('./v1');
+
+logger.info("starting superuser setup check in the background");
+
+// the following uses the V1 API to ensure that a superuser is present on the
+// currently-running instance. we might want to consider a way to do this without
+// using version-specific functionality
```

Comment:
Not if the functionality is just in `v1`. We might make some sort of global helpers that access version-specific stuff like this so that things are all in one place. Not sure.